vars/fioriOnCloudPlatformPipeline.groovy

```diff
@@ -0,0 +1,50 @@
+import static com.sap.piper.Prerequisites.checkScript
+
+import groovy.transform.Field
+
+@Field def STEP_NAME = getClass().getName()
+
+@Field def GENERAL_CONFIG_KEYS = []
+@Field def PARAMETER_KEYS = []
+@Field def STEP_CONFIG_KEYS = []
+
+/** The Scenario is intended for building and uploading a fiori application.
+  *
+  * It needs to be called from a pipeline script (Jenkinsfile) like:
+  * ```
+  *   @Library('piper-lib-os') _
+  *   @Library('your-additional-lib') __ // optional
```

Comment:
I am not sure if multiple libraries only work with providing a List of library names